helm/presets/local.yaml

```diff
@@ -1,3 +1,6 @@
+define:
+  service:
+    type: NodePort
```

Comment:
Should this be a different service type than acquire?

Reply:
this should be a `ClusterIP` type and include a path in the ingress manifest to reach Define directly from the web on 80.